src/main/java/roomescape/payment/ui/PaymentController.java

```diff
@@ -1,19 +1,32 @@
 package roomescape.payment.ui;
 
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
 import jakarta.servlet.http.HttpSession;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
-import roomescape.payment.application.dto.PrePaymentRequest;
+import roomescape.payment.application.dto.PrePaymentValidRequest;
 
+@Tag(name = "결제", description = "결제 관련 API")
 @RestController
 public class PaymentController {
 
     public static final String PRE_PAYMENT = "prePayment";
 
+    @Operation(summary = "선결제 정보 저장 API", description = "선결제 정보를 세션에 저장합니다. 결제 요청 전에 호출되어야 합니다.")
     @PostMapping("/payments")
-    public ResponseEntity<Void> setPrePayment(@RequestBody final PrePaymentRequest request, HttpSession session) {
+    public ResponseEntity<Void> setPrePayment(@RequestBody final PrePaymentValidRequest request, HttpSession session) {
```

Comment:
PrePaymentValidRequest` 요청 바디에 대한 유효성 검증을 추가해주면 조금 더 안전한 API가 되지 않을까요?  😃

```suggestion
    public ResponseEntity<Void> setPrePayment(@RequestBody @Valid final PrePaymentValidRequest request, HttpSession session) {
```

Reply:
맞습니다! 추가하겠습니당